vector/src/Data/Vector/Generic/Mutable.hs

```diff
@@ -700,21 +700,25 @@ exchange v i x = checkIndex Bounds i (length v) $ unsafeExchange v i x
 -- | Yield the element at the given position. No bounds checks are performed.
 unsafeRead :: (PrimMonad m, MVector v a) => v (PrimState m) a -> Int -> m a
 {-# INLINE unsafeRead #-}
-unsafeRead v i = checkIndex Unsafe i (length v)
+unsafeRead v !i = checkIndex Unsafe i (length v)
                $ stToPrim
```

Comment:
I 100% agree with this change, but unfortunately it means the following lines are no longer indented correctly with respect to the `=`. Is that something that can be fixed as part of this PR?